<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZapelnicFakture</class>
 <widget class="QDialog" name="ZapelnicFakture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ObliczycButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>630</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Obliczyć</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ZapisacButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>361</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisać</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>630</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Razem: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LabelRazem">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>361</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="BankLabel">
        <property name="text">
         <string>Bank: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BankLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="NumerKontaLabel">
        <property name="text">
         <string>Numer konta: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NumerKontaLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>-10</y>
     <width>20</width>
     <height>671</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>16</y>
     <width>821</width>
     <height>409</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="NumerFakturyLabel">
          <property name="text">
           <string>Numer Faktury</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NumerFakturyLineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="DataLabel">
          <property name="text">
           <string>Data:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="DzienLabel">
              <property name="text">
               <string>Dzien</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="DzienLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="MiesiacLabel">
              <property name="text">
               <string>Miesiac</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MiesiacLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="RokLabel">
              <property name="text">
               <string>Rok</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="RokLineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="SprzedawcaLabel">
          <property name="text">
           <string>Sprzedawca</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="NIPSprzedawcyLabel">
            <property name="text">
             <string>NIP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NIPSprzedawcyLineEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>268</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="ImieSprzedawcyLabel">
            <property name="text">
             <string>Imię</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ImieSprzedawcyLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="NazwiskiSprzedawcyLabel">
            <property name="text">
             <string>Nazwisko</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NazwiskoSprzedawcyLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>i/lub</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="NazwaFirmySprzedawcyLabel">
            <property name="text">
             <string>Nazwa firmy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NazwaFirmySprzedawcyLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="AdresSprzedawcyLabel">
            <property name="text">
             <string>Adres</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AdresSprzedawcyLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="NabywcaLabel">
          <property name="text">
           <string>Nabywca</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="NIPNabywcyLabel">
            <property name="text">
             <string>NIP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NIPNabywcyLineEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>268</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="ImieNabywcyLabel">
            <property name="text">
             <string>Imię</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ImieNabywcyLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="NazwiskoBabywcyLabel">
            <property name="text">
             <string>Nazwisko</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NazwiskoNabywcyLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>i/lub</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="NazwaFirmyNabywcyLabel">
            <property name="text">
             <string>Nazwa firmy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NazwaFirmyNabywcyLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="AdresNabywcyLabel">
            <property name="text">
             <string>Adres</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AdresNabywcyLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>641</width>
     <height>199</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>LP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nazwa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="LP1Edit"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="Nazwa1Box">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="LP2Edit"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="Nazwa2Box">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="LP3Edit"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="Nazwa3Box">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="LP4Edit"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="Nazwa4Box">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="LP5Edit"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="Nazwa5Box">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ilość</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="Ilosc1Edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="Ilosc2Edit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="Cena2Label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="Ilosc3Edit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="Cena3Label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="Ilosc4Edit"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="Cena4Label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="Ilosc5Edit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="Cena5Label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Cena jednostkowa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Cena1Label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>105</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Warotść towaru</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Wartosc1Label">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Wartosc2Label">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Wartosc3Label">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Wartosc4Label">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Wartosc5Label">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
